telegram/_birthdate.py

```diff
@@ -85,4 +86,4 @@ def to_date(self, year: Optional[int] = None) -> datetime:
                 "The `year` argument is required if the `year` attribute was not present."
             )
 
-        return datetime(year or self.year, self.month, self.day)  # type: ignore[arg-type]
```

Comment:
curious: what's this about?

Reply:
a new rule I added complains about datetime objects that don't have a tzinfo. I can ignore it if you like cause I think that's technically a very small breaking change.

Reply:
Ah, wait a minute: I think the real issue here is that we return a date*time* while we should return only a `date`. for datetimes it can arguably make sense that one should not use naive objects, for dates that is not the case. I'd classify this as bug on our side.

Reply:
oh wow I forgot datetime.date existed lol